backend/canvas_app_explorer/apps.py

```diff
@@ -2,4 +2,5 @@
 
 
 class CanvasAppExplorerConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
Add a comment here for why use BigAutoField, instead of the default AutoField.

Reply:
We **could** have just used AutoField but that would have required re-doing the conversions. Prior to Django 3.2 these were AutoField but you had to be explicit about it.

Reply:
I think the reason they were BigAutoField in the first place was that was what they were in the example and I didn't notice until having as the project default caused a problem. :/ It doesn't make a huge difference either way. This is just one way of specifying the default on a package rather than application level now.

Reply:
Maybe I'll just check on this with @danie1zhang and we can just re-do the migration from the start since we haven't went to production.

Reply:
I just added a migration back to AutoField. This is int(11) in the database which will be enough for all of our ids.